BUILD.bazel

```diff
@@ -127,6 +126,7 @@ selects.config_setting_group(
         "@platforms//cpu:x86_32",
```

Comment:
don't worry about 32 bit platforms, we probably will never support it

Reply:
ok, I'll remove x86 platforms